velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -260,6 +260,7 @@ TEST_F(RegexFunctionsTest, regexpReplaceSimplePosition) {
   std::string output = "Hello WorLd";
   auto result = testRegexpReplace("Hello World", "l", "L", {6});
   EXPECT_EQ(result, output);
+  EXPECT_THROW(testRegexpReplace("Hello World", "l", "L", {0}), VeloxUserError);
```

Comment:
Are you missing test cases for the new patterns being supported?

Use VELOX_ASSERT_THROW